# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/vnc/no-man.diff
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./vncviewer/Makefile	Mon Jan 20 16:53:18 2003
+++ ./vncviewer/Makefile	Mon Jan 20 17:04:35 2003
@@ -970,8 +970,6 @@
 	fi
 	$(INSTALL) $(INSTALLFLAGS) $(INSTPGMFLAGS)  vncviewer $(DESTDIR)$(BINDIR)/vncviewer
 
-all:: vncviewer.$(MANSUFFIX).html
-
 vncviewer.$(MANSUFFIX).html: vncviewer.$(MANNEWSUFFIX)
 	$(RM) vncviewer.$(MANSUFFIX).html vncviewer.$(MANSUFFIX)-html
 	$(RMAN) $(RMANOPTIONS) < vncviewer.$(MANNEWSUFFIX) \
@@ -993,13 +991,12 @@
 cleandir::
 	$(RM) vncviewer.$(MANSUFFIX).html vncviewer.$(MANSUFFIX)-html
 
-all:: vncviewer.$(MANNEWSUFFIX)
-
 vncviewer.$(MANNEWSUFFIX): vncviewer.$(MANSRCSUFFIX)
 	$(RM) $@
 	cd `dirname vncviewer` && \
 	$(LN) `basename vncviewer.$(MANSRCSUFFIX)` `basename $@`
-
+	touch vncviewer.$(MANSRCSUFFIX)
+	
 cleandir::
 	$(RM) vncviewer.$(MANNEWSUFFIX)
 
